pyg3t/gtparse.py

```diff
@@ -83,7 +161,24 @@ def _get_header(msgs):
 
 
 class Catalog(object):
-    """A Catalog represents one gettext catalog, or po-file."""
+    """A Catalog represents one gettext catalog, or po-file
+
+    Args:
+        fname (str): Filename of the source
```

Comment:
Maybe we should write "string" instead of the technical str.  In Python2, we prefer unicode to str, so ideally people should not be passing too many str.